batch/src/main/scala/uk/gov/ons/addressindex/models/HybridAddressNisraEsDocument.scala

```diff
@@ -180,6 +180,14 @@ object HybridAddressNisraEsDocument extends EsDocument {
     }
   }
 
+  def toInt(s: String): Option[Int] = {
+    try {
+      Some(s.toInt)
+    } catch {
+      case e: Exception => None
```

Comment:
This can be shortened to `Try(s.toInt).toOption`, if the exception being thrown is not a concern.